docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,507 @@
+---
+slug: manual-ci-setup
+append_help_link: true
+title: Running Semgrep in CI without Semgrep App 
+description: ""
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Running Semgrep in continuous integration (CI) without Semgrep App
+
+Run Semgrep in your Continuous Integration (CI) pipeline to scan your repository for code vulnerabilities and other issues. This guide explains how to set up Semgrep in your pipeline without the use of Semgrep App, also known as a **stand-alone** setup.
```

Comment:
Maybe we can also link to the section -  `## Migrating to Semgrep App from a manual CI setup`

Reply:
This isn't actionable right now because the "Running Semgrep CI with Semgrep App" isn't merged yet (it comes after this one.) I'll do this in another PR. I prefer not to link the migrating section, it's already in the sidebar and we're loading too many links upfront.